source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -79,6 +79,20 @@ Engine also covers the validation:
 - Physical network must be on the same Data Center as the external network.
 - The custom values (physical network and VLAN ID) must not be specified a when provider physical network is set.
 
+### Import of a provider physical network
+While importing a network with physical network access from an external network
+provider into Engine's database, Engine attempts to map this physical network has to be mapped to the corresponding logical network in Engine.
```

Comment:
attempts? beter: must
to map this physical network has to be mapped - not clear

Maybe: 
When importing a network with physical network access from an external network provider into Engine, Engine must map this physical network to the corresponding logical network in Engine.

or (with brackets to reduce sentence complexity):
When importing a network with physical network access (from external network
provider into Engine), Engine must map this physical network to the corresponding logical network in Engine.

Reply:
Done